lib/elixir/lib/range.ex

```diff
@@ -47,6 +48,12 @@ defmodule Range do
       false
 
   """
+  # @spec range?(term) :: boolean
+  @spec range?(arg) :: false when arg: %Range{first: nil, last: nil}
+  @spec range?(arg) :: true when arg: %Range{}
+  @spec range?(term) :: false 
```

Comment:
sorry Jose, I just wrote a test for that, and it doesn't break it.
https://gist.github.com/eksperimental/fc06e21d6589e0dac48f
or if you have a case, plese let  me know